boundaries/ocd-division/country:ca/province:ns/2019/definition.py

```diff
@@ -0,0 +1,17 @@
+import re
+from datetime import date
+
+import boundaries
+
+boundaries.register('Nova Scotia electoral districts',  # (2019)
```

Comment:
Also, edit the old boundaries to put (2012) in the name. See https://github.com/opennorth/represent-canada/issues/102

```suggestion
boundaries.register('Nova Scotia electoral districts (2019)',
```

Reply:
Thanks @jpmckinney -- how's that now?